client/app/components/Story/StoryActions.jsx

```diff
@@ -41,6 +41,7 @@ export type Props = {
   actions: Actions,
   hasStory?: boolean,
   dark?: boolean,
+  name?: string,
```

Comment:
I think to avoid naming collisions, we should rename this prop to `storyName` and this prop should be required, so we can remove the `?`. This also fixes the Flow error in the failed CI build. You should run `yarn lint` to reproduce it locally and verify that it's been fixed.